src/index.ts

```diff
@@ -207,6 +160,43 @@ function Loadmill(options: Loadmill.LoadmillOptions) {
         );
     }
 
+    async function _getExecutableTestSuites(): Promise<Array<Loadmill.TestSuiteDef>> {
+        const { body: { testSuites } } = await superagent.get(`${testSuitesAPI}?rowsPerPage=100&filter=CI%20enabled`)
```

Comment:
I think that you need to pass the labels too, as you might get the first 100 without any label and this is not the what the client wants to run

Reply:
Or server doesn't support it

Reply:
So lets implement it on app's server, as we might miss a lot of suites